src/oidcendpoint/endpoint.py

```diff
@@ -117,6 +117,14 @@ def construct_endpoint_info(default_capabilities, **kwargs):
                     _info[attr] = default_val
                 elif "signing_alg_values_supported" in attr:
                     _info[attr] = assign_algorithms("signing_alg")
+                    if (
+                        attr
+                        == "token_endpoint_auth_signing_alg_values_supported"
+                    ):
+                        # none must not be in
+                        # token_endpoint_auth_signing_alg_values_supported
+                        if "none" in _info[attr]:
+                            _info[attr].remove("none")
```

Comment:
sure, we could have created a default blacklist and made it more granular but ... I don't think anyone cares about "none"